tendermint/src/rpc/endpoint/commit.rs

```diff
@@ -12,9 +12,11 @@ pub struct Request {
 impl Request {
     /// Create a new request for commit info about a particular block
     pub fn new(height: block::Height) -> Self {
-        Self {
-            height: Some(height),
+        let mut height_opt = Some(height);
+        if height.value() == 0 {
+            height_opt = None
         }
+        Self { height: height_opt }
```

Comment:
I'd probably write this like:

```rust
Self {
    height: match height {
        0 => None,
        _ => Some(height)
    }
}
```

Alternatively perhaps define a method on `Height` for this? It seems useful.

Reply:
This was a shortcut to get the integration to work. There ought to be a more explicit way to request the latest height that depending on the implicit menaing of `0`.

Reply:
Addressed in 7477c9d

Reply:
Opened an issue for this https://github.com/interchainio/tendermint-rs/issues/118